papaparse.js

```diff
@@ -643,6 +644,15 @@ License: MIT
 			}
 
 			xhr.open(this._config.downloadRequestBody ? 'POST' : 'GET', this._input, !IS_WORKER);
+			if (this._config.downloadTimeout) {
+				timeout = parseInt(this._config.downloadTimeout);
+				if (timeout) {
+					xhr.timeout = timeout;
+					xhr.ontimeout = bindFunction(function() { this._chunkError('timeout'); }, this);
+				} else {
+					this._chunkError('invalid timeout');
```

Comment:
Consider helping the user here. "Why was my timeout invalid?"

Reply:
It may be enough to just tell them what they tried to use and maybe also what type that was, and what type was expected.

Reply:
Updated message to be more descriptive.